templates/mon_mysql.sql.j2

```diff
@@ -6,7 +6,7 @@ SET foreign_key_checks = 0;
 CREATE TABLE `alarm` (
   `id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL,
   `alarm_definition_id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
-  `state` enum('UNDETERMINED','OK','ALARM') COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
I'm not convinced triggers are the best option here. I would rather see a new table for the values in the enum, and a foreign key constraint referencing that new table. This makes the connection explicit and easily visible to anyone using the database, as well as modifiable without a schema update.